dist/iidxhook-18-cn.conf

```diff
@@ -0,0 +1,50 @@
+# PCBID
```

Comment:
This looks weird. Did you create copies or move the config files for 18 to 26 by accident?

Reply:
Yeah I don't know why it's showing up like that. The only change in those files is deleting the "fix BGs" option which is never checked in the iidxhooks for those styles

Reply:
<img width="354" alt="image" src="https://user-images.githubusercontent.com/1855103/231013356-967f43f6-1a9d-4568-8d06-c08824118293.png">

Look closely at the path. Your changed files are outside of `dist/iidx` for some reason. If you move these files inside `dist/iidx` that might fix it already and show correctly in the change set.

Reply:
Whoops, I must have been in the wrong directory, I'll fix that >_>